app/src/main/python/gogdl/dl/managers/task_executor.py

```
@@ -0,0 +1,759 @@
+        self.running = False
+        with self.task_cond:
+            self.task_cond.notify()
+
+        with self.temp_cond:
+            self.temp_cond.notify()
+
+        # Clean up temp directory
+        import shutil
+        try:
+            shutil.rmtree(self.temp_dir)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)